<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>832</width>
    <height>519</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <property name="margin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QFrame" name="widgetFrameOuter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="class" stdset="0">
       <string>PeripheralCollection</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout" columnstretch="2,2,5">
          <property name="leftMargin">
           <number>10</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Load Resistance</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="analysisLabel">
            <property name="text">
             <string>Analyze:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>Fit Curves</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2" rowspan="3">
           <layout class="QHBoxLayout" name="WidgetLayout" stretch="0">
            <item>
             <widget class="QFrame" name="frame_4">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <property name="margin">
                <number>0</number>
               </property>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <widget class="QDoubleSpinBox" name="resistance">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="suffix">
             <string> Ohms</string>
            </property>
            <property name="maximum">
             <double>10000.000000000000000</double>
            </property>
            <property name="value">
             <double>1000.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QPushButton" name="pushButton_2">
            <property name="text">
             <string>Save Plots</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="class" stdset="0">
          <string>PeripheralCollectionInner</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QSplitter" name="splitter">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <widget class="QFrame" name="frame_3">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>300</width>
               <height>0</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <property name="margin">
               <number>0</number>
              </property>
              <item>
               <widget class="QTableWidget" name="resultsTable">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="autoScrollMargin">
                 <number>10</number>
                </property>
                <property name="textElideMode">
                 <enum>Qt::ElideRight</enum>
                </property>
                <property name="gridStyle">
                 <enum>Qt::DashLine</enum>
                </property>
                <property name="sortingEnabled">
                 <bool>false</bool>
                </property>
                <property name="rowCount">
                 <number>50</number>
                </property>
                <property name="columnCount">
                 <number>4</number>
                </property>
                <attribute name="horizontalHeaderCascadingSectionResizes">
                 <bool>true</bool>
                </attribute>
                <attribute name="horizontalHeaderDefaultSectionSize">
                 <number>70</number>
                </attribute>
                <attribute name="horizontalHeaderMinimumSectionSize">
                 <number>70</number>
                </attribute>
                <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
                 <bool>false</bool>
                </attribute>
                <attribute name="horizontalHeaderStretchLastSection">
                 <bool>true</bool>
                </attribute>
                <attribute name="verticalHeaderStretchLastSection">
                 <bool>true</bool>
                </attribute>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <column/>
                <column/>
                <column/>
                <column/>
               </widget>
              </item>
              <item>
               <widget class="QFrame" name="frame_6">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="frameShape">
                 <enum>QFrame::NoFrame</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <item>
                  <widget class="QLabel" name="label_3">
                   <property name="text">
                    <string>Plot:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="plotAButton">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="plotBButton">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QScrollArea" name="scrollArea">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>76</width>
                <height>393</height>
               </rect>
              </property>
              <layout class="QGridLayout" name="gridLayout_4">
               <property name="margin">
                <number>2</number>
               </property>
               <property name="spacing">
                <number>2</number>
               </property>
               <item row="0" column="0">
                <layout class="QGridLayout" name="plot_area"/>
               </item>
              </layout>
             </widget>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>832</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave_as"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionSave_as">
   <property name="text">
    <string>save as</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>fit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>281</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>-19</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>plotAButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>plotA()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>492</x>
     <y>506</y>
    </hint>
    <hint type="destinationlabel">
     <x>281</x>
     <y>542</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>plotBButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>plotB()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>735</x>
     <y>506</y>
    </hint>
    <hint type="destinationlabel">
     <x>358</x>
     <y>541</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave_as</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>saveFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>415</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>savePlots()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>779</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>747</x>
     <y>-8</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>delete_curve()</slot>
  <slot>fit()</slot>
  <slot>plotA()</slot>
  <slot>plotB()</slot>
  <slot>saveFile()</slot>
  <slot>savePlots()</slot>
 </slots>
</ui>
